### PR TITLE
feature: user has default profile pictureUrl or url can be sent in re…

### PR DESCRIPTION
**Pull request by Anicaaa**
User has default profile picture url or url can be sent in the req.body

### CODE REVIEW

File: src/domain/user.js [hunk 1]
```diff
@@ -19,15 +22,23 @@ export default class User {
...
       user.role
     )
   }
 
   static async fromJson(json) {
     // eslint-disable-next-line camelcase
-    const { first_name, last_name, email, biography, github_url, password } =
-      json
+    const {
+      first_name,
```
Comment (Auenc) [inline src/domain/user.js:34]: This is causing some linting errors. Before, the variables that we were creating were all being defined on one line. So we use this comment `// eslint-disable-next-line camelcase` to tell the linter to ignore the non-camel case variables.

We now need to add that comment above each of the variables in order to stop the linting from complaining that they're not using camel-case.

**Review by Auenc — CHANGES_REQUESTED** [review]
We need to remove the console.log as well as fix the linting issue

**Review by Auenc — APPROVED** [review]
LGTM, but we might have a problem with the migrations. We will look at that when the heroku server tries to deploy the changes

**Review by shahzadnaeem — APPROVED** [review]
Approving to allow merge

**Pull request merged**